src/drawing/path.js

```diff
@@ -38,7 +39,11 @@ export default function(context, options = {}, fn) {
     context.fillStyle = fillStyle;
     context.fill();
   }
-  context.stroke();
+
+  if (lineWidth !== null) {
+    context.stroke();
+  }
```

Comment:
This should never be `null`, unless we did something wrong?

Reply:
It can be null, that's the input value, not `context.linewidth`, I read this:

```js
/*
 * @param {Number} [options.lineWidth] - The width of lines in the path. If null, no line width is set.
 * If undefined then toolStyle.getToolWidth() is set.
*/
```

as saying the outline should not be rendered if the `lineWidth` argument is null. Maybe that's not what it meant, but its a good option to have and we are using it for the segmentation fill atm.

Reply:
Perhaps we should just have a `drawLines = true` optional default parameter and make the `||`-fest above a little less confusing?

Reply:
JSDoc seems to indicate _optional_ or a number, with no default value. So it should only ever be able to be an `int` or `undefined`.

I'm okay with setting it to `0` skipping the `context.stroke();`, or adding a third parameter like `shouldDrawLines = true;` If there's a better option, don't hesitate to pursue it.

Reply:
I'll add `shouldDrawLines` to the options, and default it to true if its `undefined`.